.github/workflows/blank.yml

```diff
@@ -12,25 +12,33 @@ jobs:
         java: [11, 8]
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout repo
```

Comment:
Am nitpicking, but these actions which didn't have names are kinda self-explanatory and adding name fields just adds clutter which isn't needed